practitioner-service/src/test/java/uk/ac/ox/ndph/arts/practitioner_service/service/PersonServiceTests.java

```diff
@@ -0,0 +1,77 @@
+package uk.ac.ox.ndph.arts.practitioner_service.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.converter.ConvertWith;
+import org.hl7.fhir.r4.model.Practitioner;
+
+import org.springframework.boot.test.context.SpringBootTest;
+import static org.mockito.Mockito.when;
+import org.mockito.Mockito;
+import org.mockito.ArgumentCaptor;
+import org.junit.jupiter.api.Assertions;
+import uk.ac.ox.ndph.arts.practitioner_service.model.Person;
+import uk.ac.ox.ndph.arts.practitioner_service.service.EntityService;
+import uk.ac.ox.ndph.arts.practitioner_service.repository.FhirRepository;
+import uk.ac.ox.ndph.arts.practitioner_service.exception.HttpStatusException;
+import uk.ac.ox.ndph.arts.practitioner_service.exception.RestException;
+import uk.ac.ox.ndph.arts.practitioner_service.exception.ArgumentException;
+
+@SpringBootTest
```

Comment:
Not sure that this is required here since its not bootstrapping a spring application, just unit tests the service. 
You can also make the tests shorter by using the Mock annotations as class fields, like declaring the mock once and using it in all tests, each tests resets the stubbing you already did on the mocks:

@Mock
private FhirRepository  fhirRepository